packages/dtslint-runner/src/prepareAffectedPackages.ts

```diff
@@ -47,27 +43,15 @@ export async function prepareAffectedPackages({
   };
 }
 
-export async function installDependencies(packages: Iterable<TypingsData>, typesPath: string): Promise<void> {
+export async function installDependencies(definitelyTypedPath: string): Promise<void> {
   console.log("Installing NPM dependencies...");
   const start = Date.now();
-
-  // We need to run `npm install` for all dependencies, too, so that we have dependencies' dependencies installed.
-  for (const pkg of packages) {
-    const cwd = joinPaths(typesPath, pkg.subDirectoryPath);
-    if (!(await pathExists(joinPaths(cwd, "package.json")))) {
-      continue;
-    }
-
-    // Scripts may try to compile native code.
-    // This doesn't work reliably on travis, and we're just installing for the types, so ignore.
-    const cmd = `npm install ${npmInstallFlags} --tag ts${pkg.minTypeScriptVersion}`;
-    console.log(`  ${cwd}: ${cmd}`);
-    const stdout = await execAndThrowErrors(cmd, cwd);
-    if (stdout) {
-      // Must specify what this is for since these run in parallel.
-      console.log(` from ${cwd}: ${stdout}`);
-    }
+  const cwd = definitelyTypedPath;
+  const cmd = `pnpm install`;
```

Comment:
I suspect we shouldn't do this in the runner anymore, now that CI can do it, deal with caching, etc? Without a lockfile, this will incur the full cost of an install again, technically

Reply:
It should probably stick around to support the clone-DT-fresh-each-time scenario, but that should no longer be the default. I'm not even sure when it's useful, actually, because locally I always use my normal DT clone and don't care about `pnpm install` each time.